storage/tree/node.go

```diff
@@ -201,6 +201,23 @@ loop:
 	}
 }
 
+// NewNodeIDFromID2 constructs a NodeID from a NodeID2.
+func NewNodeIDFromID2(id NodeID2) NodeID {
+	bytes := id.FullBytes()
+	last, bits := id.LastByte()
+	path := make([]byte, len(bytes)+int(bits+7)/8)
+	copy(path, bytes)
```

Comment:
You *could* use the result of copy to determine whether the last byte needs filling in. Or not. I'm not a cop.

Reply:
I found a shorter way.